From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 07 Apr 2015 00:20:38 +0100
Subject: procfs: Avoid ABI change in 3.16.7-ckt8

genksyms is perturbed by removing this #include even though none of
its definitions are needed by exported functions...

---
--- a/fs/proc/generic.c
+++ b/fs/proc/generic.c
@@ -19,6 +19,7 @@
 #include <linux/mount.h>
 #include <linux/init.h>
 #include <linux/idr.h>
+#include <linux/namei.h>
 #include <linux/bitops.h>
 #include <linux/spinlock.h>
 #include <linux/completion.h>
